### PR TITLE
Update and fix docs (namespaces, consistency)

### PR DESCRIPTION
**Pull request by dbort**
Summary:
Audit all instances of `\bexec_aten::` and `\btorch::` under `docs/`, updating where appropriate.

The only remaining `torch::` instances are for kernels, which I didn't get a chance to migrate before v0.4.0.

Also:
- Update the LLM Manual code to be consistent between the doc and main.cpp
- Fix some LLM Manual issues: point to the latest release, and "main.cpp" instead of "main.h"

Differential Revision: D64152344

### CODE REVIEW

File: examples/llm_manual/main.cpp [hunk 1]
```diff
@@ -90,7 +90,7 @@ std::string generate(
 
 int main() {
   // Set up the prompt. This provides the seed text for the model to elaborate.
-  std::cout << "Prompt: ";
+  std::cout << "Enter model prompt: ";
```
Comment (malfet) [inline examples/llm_manual/main.cpp:93]: This change seems unrelated to PR title

File: docs/source/running-a-model-cpp-tutorial.md [hunk 2]
```diff
@@ -75,12 +86,12 @@ for (size_t id = 0; id < num_memory_planned_buffers; ++id) {
...
-// rather then a fixed buffer.
-util::MallocMemoryAllocator method_allocator;
+// Version of MemoryAllocator that uses malloc to handle allocations rather then
+// a fixed buffer.
+MallocMemoryAllocator method_allocator;
 
-// Assemble all of the allocators into the MemoryManager that the Executor
-// will use.
+// Assemble all of the allocators into the MemoryManager that the Executor will
+// use.
```
Comment (malfet) [inline docs/source/running-a-model-cpp-tutorial.md:94]: Do not seem related to the PR title

File: docs/source/running-a-model-cpp-tutorial.md [hunk 3]
```diff
@@ -75,12 +86,12 @@ for (size_t id = 0; id < num_memory_planned_buffers; ++id) {
 HierarchicalAllocator planned_memory(
     {planned_arenas.data(), planned_arenas.size()});
 
-// Version of MemoryAllocator that uses malloc to handle allocations
-// rather then a fixed buffer.
-util::MallocMemoryAllocator method_allocator;
+// Version of MemoryAllocator that uses malloc to handle allocations rather then
+// a fixed buffer.
```
Comment (malfet) [inline docs/source/running-a-model-cpp-tutorial.md:90]: Is there a linter that prefers one layout vs another?
line 89 now longer than 80 characters, isn't it?

File: docs/source/running-a-model-cpp-tutorial.md [hunk 4]
```diff
@@ -64,7 +74,8 @@ std::vector<Span<uint8_t>> planned_arenas; // Passed to the allocator
 
 size_t num_memory_planned_buffers = method_meta->num_memory_planned_buffers();
 
-// It is possible to have multiple layers in our memory hierarchy; for example, SRAM and DRAM.
+// It is possible to have multiple layers in our memory hierarchy; for example,
+// SRAM and DRAM.
```
Comment (malfet) [inline docs/source/running-a-model-cpp-tutorial.md:78]: Same as below, please don't change formatting (or mention in PR description which new linter rule mandates one to do so)

File: docs/source/running-a-model-cpp-tutorial.md [hunk 5]
```diff
@@ -48,14 +59,13 @@ One of the principles of ExecuTorch is giving users control over where the memor
...
+// Method names map back to Python nn.Module method names. Most users will only
+// have the singular method "forward".
 const char* method_name = "forward";
 
 // MethodMeta is a lightweight structure that lets us gather metadata
-// information about a specific method. In this case we are looking to
-// get the required size of the memory planned buffers for the method
-// "forward".
+// information about a specific method. In this case we are looking to get the
+// required size of the memory planned buffers for the method "forward".
```
Comment (malfet) [inline docs/source/running-a-model-cpp-tutorial.md:68]: Unrelated to PR title

File: docs/source/llm/getting-started.md [hunk 6]
```diff
@@ -309,28 +309,28 @@ penalties for repeated tokens, and biases to prioritize or de-prioritize specifi
...
+
+  // Load the exported nanoGPT program, which was generated via the previous
+  // steps.
+  Module model("nanogpt.pte", Module::LoadMode::MmapUseMlockIgnoreErrors);
+
+  const auto max_input_tokens = 1024;
+  const auto max_output_tokens = 30;
+  std::cout << prompt;
+  generate(
+      model, prompt, tokenizer, sampler, max_input_tokens, max_output_tokens);
```
Comment (malfet) [inline docs/source/llm/getting-started.md:333]: Unrelated to PR title

**Review by malfet — APPROVED** [review]
This PR contains a lots of changes unrelated to the title. If they must be landed together, please update the description

**Comment by dbort** [comment]
@malfet I mention it in the summary as "Also update the LLM Manual code to be consistent between the doc and main.cpp." Basically I had to update the namespaces in both main.cpp and in the doc that includes snippets of main.cpp, but noticed that they're different. I reformatted them and made them consistent as part of this.

**Comment by dbort** [comment]
Updated title/description because the scope of this creeped further

**Review by Olivia-liu — APPROVED** [review]
